src/wallet/rpcwallet.cpp

```diff
@@ -3787,7 +3788,14 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
     o.push_back(Pair("fee", std::stod(FormatMoney(nFee))));
     UniValue contextInfo = o;
 
+    // Builder (used if Sapling addresses are involved)
+    boost::optional<TransactionBuilder> builder;
+    if (false) { // TODO: Sapling support
+        builder = TransactionBuilder(Params().GetConsensus(), nextBlockHeight, pwalletMain);
+    }
+
     // Contextual transaction we will build on
+    // (used if no Sapling addresses are involved)
```

Comment:
I am a little confused by this. Is it the case that we are either using the TransactionBuilder or the CMutableTransaction, and never both?

Reply:
I'd like to change the function to take in a `boost::variant<TransactionBuilder, CMutableTransaction>`, but that will have to wait for #2979.